### ISSUE TITLE
description create failed after upgraded to 0.11.0

### ISSUE THREAD

**Issue by jasine** [clusternet/clusternet#422]
#### What happened:

after upgraded to 0.11.0 helm description failed to create

`kubectl describe base xxx` has following error

```
Warning  FailedApplyingOverrides   6m59s (x69 over 17m)  clusternet-hub  Failed to apply overrides for Description clusternet-452x2/dipper-helm: failed to apply OverrideConfig ignore-status: error in remove for path: '/status': strconv.Atoi: parsing "status": invalid syntax
```

#### Environment:
- Clusternet version:
    - Clusternet-agent version (user `clusternet-agent --version=json`): 0.11.0
    - Clusternet-hub version (user `clusternet-hub --version=json`): 0.11.0
- Kubernetes version (use `kubectl version`): k3s 1.24
- Cloud provider or hardware configuration: v1.23.8+k3s2
- OS (e.g: `cat /etc/os-release`): ubuntu 20.04 
- Kernel (e.g. `uname -a`): 5.4.0-109-generic
- Others:

**Comment by dixudx** [comment]
@jasine I can not reproduce this with v0.11.0.

> Events:
>  Type    Reason              Age                  From            Message
>  ----    ------              ----                 ----            -------
>  Normal  DescriptionCreated  110s                 clusternet-hub  Description clusternet-gvk7j/app-demo-helm is created successfully
>  Normal  DescriptionCreated  110s                 clusternet-hub  Description clusternet-gvk7j/app-demo-generic is created successfully
>  Normal  Synced              110s (x3 over 110s)  clusternet-hub  Base synced successfully

Actually this should had been fixed by #416.

**Comment by DanielXLee** [comment]
Also can not reproduce in TKE cluster
```
☁  .kube  helm list
NAME                	NAMESPACE        	REVISION	UPDATED                                	STATUS  	CHART                     	APP VERSION
clusternet-hub      	clusternet-system	1       	2022-07-19 07:07:15.505273121 +0000 UTC	deployed	clusternet-hub-0.7.0      	v0.11.0
clusternet-scheduler	clusternet-system	1       	2022-07-19 07:07:15.708669237 +0000 UTC	deployed	clusternet-scheduler-0.7.0	v0.11.0
☁  .kube  k describe base service-vendors -n clusternet-drcx8 | grep Events
Events:           <none>
```
